src/main/java/org/apache/sysds/utils/NativeHelper.java

```diff
@@ -416,4 +416,21 @@ public static native boolean conv2dSparse(int apos, int alen, int[] aix, double[
 	// different tradeoffs. In current implementation, we always use GetPrimitiveArrayCritical as it has proven to be
 	// fastest. We can revisit this decision later and hence I would not recommend removing this method.
 	private static native void setMaxNumThreads(int numThreads);
+
+	//test interface for native binding
+	public static native void testNativeBindingWithDgemm(char transa, char transb, int m, int n, int k, double alpha, double[] A, int lda, double[] B, int ldb, double beta, double[] C, int ldc);
```

Comment:
if you go with the separate cpp file idea, then we can move these lines to another file.